examples/tv-casting-app/tv-casting-common/core/CastingPlayerDiscovery.cpp

```diff
@@ -63,7 +63,7 @@ CHIP_ERROR CastingPlayerDiscovery::StartDiscovery(uint32_t deviceTypeFilter)
 
 CHIP_ERROR CastingPlayerDiscovery::StopDiscovery()
 {
-    ChipLogProgress(Discovery, "CastingPlayerDiscovery::StopDiscovery() mCastingPlayers: %lu, mCastingPlayersInternal: %lu",
```

Comment:
The Lint CI check is failing.

Reply:
made change to `static_cast<unsigned int>` / `%u` as was performed when adding the lint; will make sure it builds before resolving this comment.